### PR TITLE
Add script usage to the README

### PR DESCRIPTION
**Pull request by shapirone**
###  Summary
This just adds some script details to the README in case anyone comes looking for more info.

### Requirements (place an `x` in each `[ ]`)

* [x] I've read and understood the [Contributing Guidelines](https://github.com/{project_slug}/blob/main/.github/CONTRIBUTING.md) and have done my best effort to follow them.
* [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,9 +10,58 @@ This library requires a recent (at least 1.22) version of [deno](https://deno.la
...
+| `get-manifest`    | `slack manifest`       | Converts a `manifest.json`, `manifest.js`, or `manifest.ts` file int o a valid manifest JSON payload. For more information, see the [deno-slack-builder](https://github.com/slackapi/deno-slack-builder) repository's `--manifest` arg. |
+| `get-hooks`       | N/A                    | Fetches the list of available hooks for the CLI from this repository. |
+| `get-trigger`     | `slack trigger create` | Converts a specified `json`, `js`, or `ts` file into a valid trigger JSON payload to be uploaded by the CLI to the `workflows.triggers.create` Slack API endpoint. |
+| `install-update`  | `slack upgrade`        | Prompts the user to automatically update any dependencies that need to be updated based on the result of the `check-update` hook. |
+| `start`           | `slack run`            | Creates a socket connection between the Slack CLI and a Slack workspace for local development that includes hot reloading. For more information, see the [deno-slack-runtime](https://github.com/slackapi/deno-slack-runtime) repository's details on `local-run`. |
```
Comment (filmaj) [inline README.md:24]: Hate to be pedantic, but this is not technically correct - the CLI manages the connection for deno apps. Here's my suggestion - feel free to ignore as well.

```suggestion
| `start`           | `slack run`            | While developing and locally running a deno-slack-based application, the CLI manages a socket connection with Slack's backend and delegates to this hook for invoking the correct application function for relevant events incoming via this connection. For more information, see the [deno-slack-runtime](https://github.com/slackapi/deno-slack-runtime) repository's details on `local-run`. |
```

File: README.md [hunk 2]
```diff
@@ -10,9 +10,58 @@ This library requires a recent (at least 1.22) version of [deno](https://deno.la
...
+
+### Check Update Script Usage
+The `check_update.ts` file is executed as a Deno program and takes no arguments.
+
+#### Example
+```bash
+deno run -q --config=deno.jsonc --allow-read --allow-net https://deno.land/x/deno_slack_hooks/check_update.ts
+```
+### Get Hooks Script Usage
+The `mod.ts` file is executed as a Deno program and takes one no arguments
```
Comment (filmaj) [inline README.md:35]:
```suggestion
The `mod.ts` file is executed as a Deno program and takes no arguments.
```

File: README.md [hunk 3]
```diff
@@ -10,9 +10,58 @@ This library requires a recent (at least 1.22) version of [deno](https://deno.la
...
+| `--source` | Absolute or relative path to your target trigger file. The trigger object must be exported as default from this file. |
+
+
+#### Example
+```bash
+deno run -q --config=deno.jsonc --allow-read --allow-net https://deno.land/x/deno_slack_hooks/get_trigger.ts --source="./trigger.ts"
+```
+
+### Install Update Script Usage
+The `check_update.ts` file is executed as a Deno program and takes no arguments.
```
Comment (filmaj) [inline README.md:56]:
```suggestion
The `install_update.ts` file is executed as a Deno program and takes no arguments.
```

**Review by filmaj — APPROVED** [review]
LGTM, left a few suggestions.

File: README.md (hunk 1 above)
Comment (shapirone) [reply README.md:24]: So the CLI would treat non-Deno apps differently if they implemented the same logic that's in our hook?

Comment (filmaj) [reply README.md:24]: Yes, the details for this hook are subtle but that's true. Bolt, for example, handles the websocket connection, so the CLI doesn't need to worry about it. Check out the CLI/SDK interface spec quip doc, under the 'list of hooks', you'll see two entries for the `start` hook. The difference is what the value of the config field in the `slack.json` is (`managed_connection: true` vs. `managed_connection: false`)

Comment (filmaj) [reply README.md:24]: You can also take a look and [see how the CLI acts differently based on the value of this `managed_connection` configuration](https://github.com/slackapi/slack-cli/blob/main/pkg/cloud/run.go#L134-L153).

**Pull request merged**